decisions/Gondor.txt

```diff
@@ -1911,14 +1912,12 @@ political_decisions = {
 					UMB = { exists = no }
 				}
 			}
-			NOT = {
-				UMB_256 = { NOT = { is_core = GON } }
-				UMB_260 = { NOT = { is_core = GON } }
-				UMB_263 = { NOT = { is_core = GON } }
-				UMB_270 = { NOT = { is_core = GON } }
-				UMB_273 = { NOT = { is_core = GON } }
-				HAR_277 = { NOT = { is_core = GON } }
-			}
+			UMB_256 = { NOT = { is_core = GON } }
+			UMB_260 = { NOT = { is_core = GON } }
+			UMB_263 = { NOT = { is_core = GON } }
+			UMB_270 = { NOT = { is_core = GON } }
+			UMB_273 = { NOT = { is_core = GON } }
+			HAR_277 = { NOT = { is_core = GON } }
```

Comment:
Revert this. The original allows you to take the decision if any province in the regions is not a core. With what you have makes it so you can only take the decision if all provinces are not cores

Reply:
The original prevented the decision from showing up at all.

Reply:
Damn you're right -- I obviously messed up when logicing it out originally. I did some testing locally, and you'll want to replace that outer NOT with an OR. Removing the NOT completely makes it so if Gondor has cores on an entire region there, the decision won't show up, even if they're missing a core somewhere

Reply:
How about making it 
`NOT = {
	OR = {
		UMB_256 = { is_core = GON  }
		UMB_260 = { is_core = GON }
		UMB_263 = { is_core = GON  }
		UMB_270 = { is_core = GON  }
		UMB_273 = { is_core = GON  }
		HAR_277 = { is_core = GON }
	}
}`

Reply:
Why?

Reply:
Putting the NOT = {...} inside the states is kinda messy jand inelegant IMHO

Reply:
I do agree, however, switching where the NOT is unfortunately changes the logic due to how state-level checks work...

UMB_256 = { is_core = GON  } will return true if any province in UMB_256 is a core of Gondor
NOT = { UMB_256 = { is_core = GON } } will return true if all provinces in UMB_256 are not cores of Gondor
UMB_256 = { NOT = { is_core = GON } } will return true if any province in UMB_256 is not a core of Gondor (this is the one that we want to be checking for)

Reply:
Ok then. I'll leave it as it is.